README.rst

```diff
@@ -21,6 +21,8 @@
 Edalize
 =======
 
+Find the docs at `edalize.rtfd.io <https://edalize.rtfd.io>`__.
```

Comment:
Can you put this comment in the beginning of the section "How to use it?". And docs -> documentation.

Reply:
Done, by the way, I added a textual ref to this repo in section "Install it".